src/components/List.js

```diff
@@ -3,7 +3,13 @@ import ListItem from './ListItem';
 import PlaceholderItem from './PlaceholderItem';
 import PropTypes from 'prop-types';
 
-const List = props => {
+const List = (props) => {
+  const [canDrag, setCanDrag] = React.useState(true);
+
+  const updateCanDrag = (flag) => {
+    setCanDrag(flag);
+  };
```

Comment:
Why is this method necessary?  I'm curious if passing it does anything different than passing `setCanDrag` as a prop to `ListItem`.

Reply:
Yeah it should work like that, but when I first tried that it didn't work. Maybe it was because of something else. I'll try it :+1:

Reply:
The way passed the hooks method was wrong, that's why it didn't work before.